Lighty/LightBox.cs

```diff
@@ -292,7 +292,7 @@ protected async Task RemoveDialogAsync(FrameworkElement dialog)
 
             if (this.IsParallelDispose)
             {
-                this.DestroyDialogAsync(dialog);
+                var _ = this.DestroyDialogAsync(dialog);
```

Comment:
```suggestion
                this.DestroyDialogAsync(dialog);
```

Reply:
The changes of this line is not needed.

Reply:
I added this because there are some warnings with VS2019, IIRC. And _ is the "discard" operator.
It complains because a task is launched and no reference is held to that task, and that's not supposed to happen

Reply:
Oh, I See 😃 
Thaks for fix it.
I'll merge it, and release as FluentWPF v0.3.2 soon.
Please wait for a while.